spaces/aliases.json

```
@@ -23,5 +23,6 @@
 	"pepemon": "pepedontdump.eth",
 	"akropolis-delphi": "akropolis.eth",
 	"rally": "rallygov.eth",
-	"debaseonomics": "debaseonomics.eth"
+	"debaseonomics": "debaseonomics.eth",
+	"spookyswap": "spookyswap.eth"
```

When creating a space you do not have to edit the `aliases.json` file.
```suggestion
	"debaseonomics": "debaseonomics.eth"
```